roles/StackStorm.st2smoketests/tasks/main.yml

```
@@ -41,12 +41,14 @@
   tags:
     - smoke-tests
 
-- name: Install st2 pack from exchange
-  become: yes
+- name: Get st2 pack info
   environment:
     ST2_AUTH_TOKEN: "{{ st2_token.stdout }}"
-  command: "st2 pack install st2"
+  command: st2 pack get st2 -j
```

:+1:

The only question: is there a good reason to output as json? 
```suggestion
  command: st2 pack get st2
```

---

agreed.